server/proxy/proxy.go

```diff
@@ -61,43 +61,77 @@ func (p *Proxy) Call(in Incoming, creq apps.CallRequest) CallResponse {
 		return NewProxyCallResponse(apps.NewErrorResponse(err))
 	}
 
-	cresp := p.callApp(in, *app, creq)
+	cresp, _ := p.callApp(in, *app, creq)
 	return NewProxyCallResponse(cresp).WithMetadata(AppMetadataForClient{
 		BotUserID:   app.BotUserID,
 		BotUsername: app.BotUsername,
 	})
 }
 
-func (p *Proxy) callApp(in Incoming, app apps.App, creq apps.CallRequest) apps.CallResponse {
+func (p *Proxy) call(in Incoming, app apps.App, call apps.Call, cc *apps.Context, valuePairs ...interface{}) apps.CallResponse {
```

Comment:
0/5 maybe it's time to move this to `server/proxy/call.go`

Reply:
👍  another PR.